.env.development

```diff
@@ -20,3 +20,4 @@ NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=7480e74780d20eb6db1056eab0de6ddb
 NEXT_PUBLIC_COOKIE3_SITEID=405
 NEXT_PUBLIC_LIFI_API_KEY=
 NEXT_PUBLIC_JUMPER_API=https://r1kk8tftf9.execute-api.eu-central-1.amazonaws.com
+NEXT_REVALIDATION_SECRET=
```

Comment:
@oktapodia 
- not naming it "PUBLIC" as it´s not
- leaving it empty, because it can be set via vercel, right? just like lifi-api-key